src/lib/utilities/schedule-frequency-formatting.ts

```diff
@@ -0,0 +1,209 @@
+import { monthNames, weekDays } from './calendar';
+import type { ScheduleRange, StructuredCalendar } from '$lib/types/schedule';
+
+// Examples of output
+// Wednesday at 06:30pm UTC
+// Sunday, Monday, Tuesday, Wednesday, Thursday, Friday at 12:00pm UTC
+// Every 1, 15, 31 of the month at 12:00pm UTC
+// Every 1, 15, 31 of January, March, April, June, July, September, December at 12:00pm UTC
+
+const isDailyForTheMonth = (calendar: StructuredCalendar) => {
+  return (
+    calendar.dayOfMonth?.length === 1 &&
+    calendar.dayOfMonth[0]?.start === 1 &&
```

Comment:
Looking at the protos it seems like all the properties are optional and can be `null` for  `StructuredCalendar`.  If so, we might want this instead
```suggestion
    calendar.dayOfMonth?.[0]?.start === 1 &&
```

Although, TypeScript seems fine with it so maybe I'm misinterpreting 🤔

Reply:
I have the calendar.dayOfMonth?.length === 1 check before that line

Reply:
🤦‍♀️ that you do.